products/mozilla_org/extract_strings.md

```diff
@@ -0,0 +1,79 @@
+# Process overview to expose mozilla.org content for localization
+
+This document describes the tools and processes involved in converting a mozilla.org template in English to a .lang file containing localizable content and relevant information to help with localization.
+
+## This is a brief overview of the process
+
+* In [Bedrock](https://github.com/mozilla/bedrock), when a page is in the final coding review phase in GitHub and content is finalized, an [L10n label](https://github.com/mozilla/bedrock/pulls?q=is%3Aopen+is%3Apr+label%3AL10N) is added to the PR, signaling that it is ready for string extraction.
+* Strings are extracted from the identified templates, generating a temporary .lang file.
```

Comment:
A new line like this doesn't have any practical effect in Markdown. You should just continue with `The new strings…` on the same line at this point.